examples/dynamic-bridge-app/linux/include/GeneratedClusters.h

```diff
@@ -24,4 +24,16 @@ class GeneratedCluster
     virtual chip::ClusterId GetClusterId() = 0;
     // Gets the list of available attributes for this cluster.
     virtual std::vector<AttributeInterface *> GetAttributes() = 0;
+
+    // Returns a list of client to server commands. Can be nulttptr or terminated by 0xFFFF_FFFF.
```

Comment:
s/nulttptr/nullptr

Reply:
Fixed in [4496927](https://github.com/project-chip/connectedhomeip/pull/23840/commits/449692743386c4eee4a1a8a5292b3b970ac1aeb9)